src/extensions/lens-extension.ts

```diff
@@ -113,7 +113,7 @@ export class LensExtension {
   }
 
   @action
```

Comment:
This does not need this annotation, we should remove it.

Reply:
Removed